### PR TITLE
Going full bignumber

### PR DESCRIPTION
**Pull request by EyeOfPython**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,6 @@
     "svg": "cd src/svgs/ && node generate-svg-sprite.js && cd ../../ && node copy-svgs.js",
     "prod": "node clean-prod.js && ng build --aot --prod --build-optimizer --output-path=prod/ --delete-output-path=false && node copy-index.js",
     "push-prod": "cd prod && git add . && git commit -m \"build(slpdev): release\" && git push",
-    "release": "npm run prod && npm run push-prod",
```
Comment (Chippi) [inline package.json]: @EyeOfPython Please keep this one :)

File: package.json [hunk 2]
```diff
@@ -31,21 +30,21 @@
...
     "qrcode": "^1.3.3",
     "rxjs": "~6.5.2",
     "simplebar": "^4.0.0",
-    "slpdex-market": "0.1.10",
+    "slpdex-market": "^0.1.13",
     "tslib": "^1.9.0",
     "zone.js": "~0.9.0"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.800.1",
```
Comment (Chippi) [inline package.json]: Also keep this versions pls

File: package.json (hunk 1 above)
Comment (EyeOfPython) [reply package.json]: Oops! Was totally intentional to check if you're actually reviewing ;)

Comment (Chippi) [reply package.json]: No worries :)

File: src/app/helpers.ts [hunk 3]
```diff
@@ -1,12 +1,12 @@
 import BigNumber from 'bignumber.js';
 import QRCode from 'qrcode';
 
-export const convertSatsToBch = (sats: number) => {
-  return +new BigNumber(sats).div(100000000).toFormat(8);
+export const convertSatsToBch = (sats: BigNumber) => {
+  return sats.div(100_000_000);
```
Comment (Chippi) [inline src/app/helpers.ts]: @EyeOfPython Wait is 100_100_100 a BigNumber feature? You can't do this normally right 😅.

File: src/app/pipe/bch-separator.pipe.ts [hunk 4]
```diff
@@ -6,11 +6,11 @@ import BigNumber from 'bignumber.js';
 })
 export class BchSeparatorPipe implements PipeTransform {
   transform(value: string, args?: any): any {
-    if (!value) {
+    if (!value || value == 'n/a') {
```
Comment (Chippi) [inline src/app/pipe/bch-separator.pipe.ts]: @EyeOfPython Should always be triple equals ===

**Comment by Chippi** [comment]
image

Nice work, LGTM.

Just a few formatting errors. 

If haven't already, I recommend you to install these extensions. They will help you catch any formatting / lint error. Prettier will format files using shared formatting settings.

- TSLint
- Prettier

File: src/app/helpers.ts (hunk 3 above)
Comment (EyeOfPython) [reply src/app/helpers.ts]: you can do that in TypeScript but not in JavaScript :P

Comment (Chippi) [reply src/app/helpers.ts]: Good to know 👍

**Pull request merged**